uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -106,6 +112,7 @@ public static UiObject obtainEnabledUiObjectWithExactText(@NonNull final String
      * @return the UiObject associated to the supplied text
      */
     public static UiObject obtainUiObjectWithText(@NonNull final String text) {
+        Logger.i(TAG, "Obtain an instance of the UiObject for the given text..");
```

Comment:
let's also log the text in the log message

Reply:
added.